modules/tinymce/src/core/test/ts/browser/fmt/FontInfoTest.ts

```diff
@@ -138,12 +139,14 @@ UnitTest.asynctest('browser.tinymce.core.fmt.FontInfoTest', function (success, f
     iframe.contentDocument.close();
   });
 
-  suite.test('comments should always return empty string', function () {
-    assertComputedFontProp('fontFamily', '<!-- comment -->', [ 0 ], '');
-    assertComputedFontProp('fontSize', '<!-- comment -->', [ 0 ], '');
-    assertSpecificFontProp('fontFamily', '<!-- comment -->', [ 0 ], '');
-    assertSpecificFontProp('fontSize', '<!-- comment -->', [ 0 ], '');
-  });
+  if (!Env.browser.isIE()) {
```

Comment:
Why was this added? Does this mean it doesn't work on IE/returns different results? This smells like a potential regression/bug.